jupyterhub/app.py

```diff
@@ -606,6 +606,31 @@ def _authenticator_default(self):
         """
     ).tag(config=True)
 
+    throttle_retry_suggest_min = Integer(
+        30,
+        help="""
+        Minimum seconds after which we suggest the user retry spawning.
```

Comment:
Smallest period (floor) in seconds to suggest a user wait before retrying.

When `concurrent_spawn_limit` is exceeded, spawning is throttled. We
suggest users wait a random period of time, bounded by the floor and ceiling,
before retrying.

The floor should ideally be set to the median spawn time of your installation.